src/reverse/RED4ext/ResourceDepot.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
```

Comment:
Put this header in src/reverse, no need for new subfolder

Reply:
Also, we use .h for headers, would not mix header extensions rather

Reply:
Renamed file